Makefile

```diff
@@ -218,3 +218,7 @@ fuzz-native:
 	KUBEBUILDER_ASSETS=$(KUBEBUILDER_ASSETS) \
 	FUZZ_TIME=$(FUZZ_TIME) \
 		./tests/fuzz/native_go_run.sh
+
+cluster:
+	-k3d registry create flux-registry --port 58034
+	k3d cluster create flux --registry-use k3d-flux-registry:58034 --wait
```

Comment:
Ah that was a leftover from when I had the `.devcontainer.json` committed. Was only ever meant for my own personal local development environment. I've since removed it in [6ab1483](https://github.com/fluxcd/source-controller/pull/1075/commits/6ab1483652bf299413a1d508eb6da2fcc4738bf9).